packages/libs/react-ui/src/components/Select/Select.tsx

```diff
@@ -22,6 +25,7 @@ export interface ISelectProps
   onChange: React.ChangeEventHandler<HTMLSelectElement>;
   ref?: React.ForwardedRef<HTMLSelectElement>;
   ariaLabel: string;
+  variant?: SelectVariants;
```

Comment:
If you write it this way, you can remove the SelectVariants type
```suggestion
  variant?: keyof typeof selectVariants;
```